assets/css/structures/_header.scss

```diff
@@ -19,17 +19,17 @@
   // cap the max-height of the title
   // at the same size that 12vh computes to
   // when the viewport is 850px high
-  .title__thicc {
+  .title--thicc {
     font-size: 6.75rem;
   }
 }
 
 @media (min-width: #{pxToRem(800)}) {
-  .title__thicc {
+  .title--thicc {
     grid-column: 1 / 3;
   }
 
-  .title__thicc + p {
+  .title--thicc + p {
```

Comment:
These styles make the class hard to reuse. Maybe add a class `inctroduction` to the homepage and scope these to `introduction__title` and `inctroduction__intro` or something similar.

Reply:
heartily agree. I will have bandwidth this weekend, I'll take a moment to fix them all

Reply:
thanks for the thorough review ovlb!

Reply:
Great, thanks :)